src/cuga/backend/server/main.py

```
@@ -2599,6 +2605,120 @@ async def save_policies_config(
+        except Exception as sync_exc:
+            logger.warning(f"Failed to sync policy to config_store: {sync_exc}")
+            # Don't fail the request if sync fails - the policy is already saved to policy storage
+
+        return JSONResponse(result, status_code=200)
+    except ValueError as exc:
+        return JSONResponse({"status": "error", "message": str(exc)}, status_code=400)
+    except LookupError as exc:
+        return JSONResponse({"status": "error", "message": str(exc)}, status_code=404)
+    except TypeError as exc:
+        return JSONResponse({"status": "error", "message": str(exc)}, status_code=400)
+    except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)